models/LatestPostsSection.yaml

```diff
@@ -1,6 +1,6 @@
 type: object
-name: PostFeedSection
```

Comment:
@TomasBankauskas please leave the name of the component and model as PostFeedSection. You can rename the label tho.